### PR TITLE
Bug Report: Documentation Typo

### LINKED ISSUE #426: Bug Report: Documentation Typo
**Describe the bug**
I've noticed some documentation typos on the About Us and Core Functionalities Page

Screenshot (33)

Screenshot (32)

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Yass32**
**What kind of change does this PR introduce?**

Bugfix: Fixed typos on about.md and core-functionalities.md

**Issue Number:**

Fixes #426 

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**

### CODE REVIEW

File: docs/about.md [hunk 1]
```diff
@@ -19,9 +19,13 @@ Calico is modeled on the Google Summer of Code. In 2021 The Palisadoes Foundatio
...
 ​
-Though software applications exist for these types of organizations (eg. Breeze ChMS, The Realm) they assume their memberships are fully literate and have access to email. We wanted to create a system that would work for countries with similar technology and education challenges as Jamaica.
+Though software applications exist for these types of organizations (eg. Breeze ChMS, The Realm) they assume their memberships are fully literate and have access to email. We wanted to create a system that would work for countries with similar technology and education challenges as Jamaica. 
 ​
-Talawa has three main components. A mobile application with social media features, a web based portal to be used by the organization’s administrative team, and finally an API providing access to data and features.
+Talawa has three main components: 
+* A mobile application with social media features
+* A web based portal to be used by the organization’s administrative team 
+* An API providing access to data and features
```
Comment (salman-farooqi) [inline docs/about.md]: In my opinion (I maybe am wrong):
```suggestion
* A mobile application with social media features.
* A web based portal to be used by the organization’s administrative team, and 
* An API providing access to data and features.
```

File: docs/functionalities/core-functionalities.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ These are scheduled events managed by coordinators and administrators of an orga
     1. Create tasks for any of the event roles
     2. Set deadlines for completing assigned tasks
 6. User can view the event and  project details
-    1. Automatic group chat is created for persons within an event
+    1. An Automatic group chat is created for persons within an event
```
Comment (Cioppolo14) [inline docs/functionalities/core-functionalities.md]: Please remove the capital letter from "Automatic" so it is "An automatic"

File: docs/about.md (hunk 1 above)
Comment (Cioppolo14) [inline docs/about.md]: Please incorporate salman-farooqi's changes below.

**Review by Cioppolo14 — CHANGES_REQUESTED** [review]
Please resolve the comments below and ensure there are no changes in the index.html.  Thank you!

**Comment by Yass32** [comment]
@Cioppolo14 its done, and no i haven't made any changes to index.html

**Comment by Cioppolo14** [comment]
> @Cioppolo14 its done, and no i haven't made any changes to index.html

Do you know why the index.html is in your "files changed" list? Can you delete it?

File: docs/about.md [hunk 3]
```diff
@@ -19,9 +19,13 @@ Calico is modeled on the Google Summer of Code. In 2021 The Palisadoes Foundatio
 
 Talawa was created to help community based organizations collaborate with their membership. These organizations would include religious groups, non-profit charities, social groups and in limited cases, businesses.
 ​
-Though software applications exist for these types of organizations (eg. Breeze ChMS, The Realm) they assume their memberships are fully literate and have access to email. We wanted to create a system that would work for countries with similar technology and education challenges as Jamaica.
+Though software applications exist for these types of organizations (eg. Breeze ChMS, The Realm) they assume their memberships are fully literate and have access to email. We wanted to create a system that would work for countries with similar technology and education challenges as Jamaica. 
 ​
-Talawa has three main components. A mobile application with social media features, a web based portal to be used by the organization’s administrative team, and finally an API providing access to data and features.
+Talawa has three main components: 
+* A mobile application with social media features.
```
Comment (Cioppolo14) [inline docs/about.md]: Please change the period after "social media features" to a comma.

**Review by Cioppolo14 — CHANGES_REQUESTED** [review]
Please address the comment for about.md and the index.html issue.

**Comment by Cioppolo14** [comment]
@Yass32 Any progress on this?

**Comment by Yass32** [comment]
> @Yass32 Any progress on this?

Yes done

**Review by Cioppolo14 — CHANGES_REQUESTED** [review]
Please ensure that there are no file changes in the index.html file.

**Comment by Yass32** [comment]
> Please ensure that there are no file changes in the index.html file.

Ive removed it. is that okay?

**Review by Cioppolo14 — CHANGES_REQUESTED** [review]
Please remove the styles.css file as well.

**Comment by Yass32** [comment]
> Please remove the styles.css file as well.

its done

**Review by Cioppolo14 — APPROVED** [review]

**Pull request merged**